src/components/Wrap/index.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx jsx */
+import * as React from "react";
+import { jsx, css } from "@emotion/core";
+import { Theme } from "../../types/global";
+
+type MaxWidths = 1 | 2 | 3;
+
+interface Props {
+  size?: MaxWidths;
+}
+
+const Wrap: React.FC<Props> = ({ size = 1, ...props }) => {
```

Comment:
I think should we through in the `as` prop here so that we can define this as a `main` or `footer` or whatever down the line/